3-plot_category_kappa.R

```diff
@@ -43,28 +41,82 @@ array.files <- lf[grepl(paste0(file_identifier,
                                "_train_3_models_array_kappa_"), lf)]
 seq.files <- lf[grepl(paste0(file_identifier,
                              "_train_3_models_seq_kappa_"), lf)]
+if (null_model) {
+  null_array.files <- lf[grepl(paste0(file_identifier,
+                                      "_null_train_3_models_array_kappa_"), lf)]
+  null_seq.files <- lf[grepl(paste0(file_identifier,
+                                    "_null_train_3_models_seq_kappa_"), lf)]
+  
+  # check that we have ordered pairs of regular and null files for array and seq
+  array_seeds <- stringr::str_sub(array.files, -8, -5)
+  null_array_seeds <- stringr::str_sub(null_array.files, -8, -5)
+  seq_seeds <- stringr::str_sub(seq.files, -8, -5)
+  null_seq_seeds <- stringr::str_sub(null_seq.files, -8, -5)
```

Comment:
Agreed. Not sure what the best solution is. One mitigation of this is we have previously forced the seed to be a four digit number, so as long as it's the last thing to come before `.tsv`, it will be captured here.